src/main/java/com/minecolonies/coremod/util/WorkerUtil.java

```diff
@@ -165,11 +165,11 @@ public static boolean setSpawnPoint(@Nullable final BlockPos spawnPoint, @NotNul
      * @param blockHardness the hardness.
      * @return the toolType to use.
      */
-    public static IToolType getBestToolForBlock(final BlockState state, float blockHardness)
+    public static IToolType getBestToolForBlock(final BlockState state, float blockHardness, final boolean canUseShears)
```

Comment:
I thought about the implementation of this, but this is very inflexible.
a) Hand the worker building through
b) Have only one global setting for this (store the key in the AbstractBuilding)
c) Here we then query getOptionalSetting with the shears key from the building